src/components/layout/MainNavigation.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useContext, useState } from "react"
+import AppBar from "@material-ui/core/AppBar"
+import Toolbar from "@material-ui/core/Toolbar"
+import Typography from "@material-ui/core/Typography"
+import IconButton from "@material-ui/core/IconButton"
+import MenuIcon from "@material-ui/icons/Menu"
+import AccountCircle from "@material-ui/icons/AccountCircle"
+import Switch from "@material-ui/core/Switch"
+import FormControlLabel from "@material-ui/core/FormControlLabel"
+import FormGroup from "@material-ui/core/FormGroup"
+import MenuItem from "@material-ui/core/MenuItem"
+import Menu from "@material-ui/core/Menu"
+import styled from "styled-components"
+import { Grid } from "@material-ui/core"
+import AuthContext from "../../contexts/AuthProvider"
+import { useHistory } from "react-router"
+
+export const MainNavigation: React.FC = (props) => {
```

Comment:
```suggestion
export const Navbar: React.FC = (props) => {
```
đồng thời rename thành Navbar luôn
bỏ file này ở ngoài components luôn

Reply:
props component nên define type và export